packages/fe-fpm-writer/src/page/object.ts

```diff
@@ -12,9 +13,14 @@ import type { ObjectPage, InternalObjectPage } from './types';
  */
 function enhanceData(data: ObjectPage, manifest: Manifest): InternalObjectPage {
     const config: InternalObjectPage = { settings: {}, ...data, name: 'ObjectPage', ...getFclConfig(manifest) };
+    const minVersion = coerce(data.minUI5Version);
 
     // move settings into correct possition in the manifest
-    config.settings.entitySet = data.entity;
+    if (minVersion && minVersion.minor >= 94) {
```

Comment:
same here, missing check for major version. Perhaps: 
```suggestion
    if (minVersion && (minVersion.major > 1 || minVersion.minor >= 94)) {
```

Reply:
I have used gt(e) now